### DISCUSSION TITLE
Discontinued?

### DISCUSSION THREAD

**Discussion by 0xspade** [simple-login/app#1917] [Self-hosting question]
Hi @nguyenkims 

Just want to ask question. Since proton already bought Simple-login, does it mean that the self-hosting of simple-login is discontinued?

The last update was 10 months ago. Are there any chance that we the guys who self-host simple-login can get a further updates?

**Comment by nguyenkims** [comment]
@0xspade Hi, no the self hosting version is still maintained. Actually a lot of users just run on the latest version v4.35.2 which is used on app.simplelogin.io without any issue. We don't put this version in the documentation as we haven't tested it in a self hosted setup yet.

**Reply by 0xspade** [reply]
@nguyenkims May I know how to upgrade to the current build?

**Reply by nguyenkims** [reply]
We have a todo to update the self hosting instruction to the latest version which is currently running on app.simplelogin.io
You can also first upgrade to simplelogin/app:4.6.5-beta with the instructions on https://github.com/simple-login/app/tree/new-self-host-version then upgrade to the latest version `simplelogin/app-ci:v4.36.6`

**Reply by RequestPrivacy** [reply]
What commands would be involved for updating from `simplelogin/app:4.6.5-beta` to `simplelogin/app-ci:v4.36.6`? I tried to pull the later and run the sl-job-runner, sl-email, sl-app (and sl-db) commands analogue to the 4.6.5-beta ones but then after log-in it shows me a server error?

EDIT:
I forgot to start a new db container and run the migration and init docker containers on it afterwards. So docker run each: postgres -> migration -> init -> sl-app -> sl-email -> sl-job-runner and it works

**Reply by springcomp** [reply]
@RequestPrivacy  for the record, [here](https://github.com/springcomp/self-hosted-simplelogin/tree/4.36.6?tab=readme-ov-file#how-to-upgrade) are the instructions you need.
